recipes/galera.rb

```diff
@@ -67,13 +67,18 @@
     package 'rsync' do
       action :install
     end
-  elsif node['mariadb']['galera']['wsrep_sst_method'] =~ /^xtrabackup(-v2)?/
+  elsif node['mariadb']['galera']['wsrep_sst_method'] =~ /^xtrabackup(-v2)?$/
     %w(percona-xtrabackup socat pv).each do |pkg|
       package pkg do
         action :install
       end
     end
-  end
+  elsif node['mariadb']['galera']['wsrep_sst_method'] == 'xtrabackup-v24'
+    %w(percona-xtrabackup-24 socat pv).each do |pkg|
+      package pkg do
+        action :install
+      end
+    end
```

Comment:
There is an `end` missing here, if you could add it then tests should pass.

Reply:
Thanks for spotting that out. It's fixed now. Please (re-)review it.